src/main/scala/tech/cryptonomic/conseil/util/DatabaseUtil.scala

```diff
@@ -101,7 +101,7 @@ object DatabaseUtil {
         */
       def addGroupBy(aggregation: Option[Aggregation], columns: List[String]): SQLActionBuilder = {
         val columnsWithoutAggregation = columns.filterNot(col => aggregation.exists(_.field == col))
-        if(columnsWithoutAggregation.isEmpty) {
+        if(aggregation.isEmpty) {
```

Comment:
If the check is now on the optional argument I think we can make this a lot simpler
```scala
aggregation.fold(action) {
  aggregates =>
    val col... = columns.filterNot(_ == aggregates.field)
    concatenateSqlActions(....)
}
```

Reply:
changed, I haven't thought of that :/